templates/gitlab-ci-template-cloud-run.yml

```diff
@@ -71,12 +73,12 @@ workflow:
       else
         export SERVICE_ACCOUNT_OPT=""
       fi
-    # If the DEPLOYMENT_SUFFIX variable is not defined, the FULL_DEPLOYMENT_NAME is equal to DEPLOYMENT_NAME otherwise the suffix will be appended.
+    # If the DEPLOYMENT_SUFFIX variable is not defined, the DEPLOYMENT_FULL_NAME is equal to DEPLOYMENT_NAME otherwise the suffix will be appended.
```

Comment:
You know what, would be very nice if this kind of information to be printed on the pipeline too.

Reply:
it could be something like a help() function but always printed on the pipeline log, what do you think ?

Reply:
it is already implemented with [this](https://github.com/sparkfabrik/spark-k8s-deployer/blob/master/templates/gitlab-ci-template-cloud-run.yml#L82) template used [here](https://github.com/sparkfabrik/spark-k8s-deployer/blob/master/templates/gitlab-ci-template-cloud-run.yml#L108) and [here](https://github.com/sparkfabrik/spark-k8s-deployer/blob/master/templates/gitlab-ci-template-cloud-run.yml#L124).